src/base/static/components/organisms/map-legend-panel.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Image } from "../atoms/imagery";
+import { Paragraph, Header4 } from "../atoms/typography";
+import MapLegendGroup from "../molecules/map-legend-group";
+
+import "./map-legend-panel.scss";
+
+const MapLegendPanel = props => {
+  return (
+    <div className="map-legend-panel">
```

Comment:
Can we refactor this to use CSS-in-JS as well? I would like for all of our new components to be written in CSS-in-JS.